intervention_compassion/data/gmc_action.xml

```diff
@@ -89,6 +89,30 @@
             <field name="auto_process">1</field>
         </record>
 
+        <!-- MileStonesMessage -->
+        <record id = "Intervention_reporting_milestone" model = "gmc.action">
```

Comment:
Please avoid the spaces around the attributes : `id="` instead of `id = "`. Correct it in all xml